docs/guide-deploy-rpm.md

```diff
@@ -0,0 +1,206 @@
+- minio
+- tomcat
+- molgenis
+- httpd
+- *optional* opencpu
+  - opencpu-server
+  - rapache
+- *optional* python3
+
+You can do this by typing this command: ```yum install #package name#```.
+
+> note: the order of the packages is important. We do no have requirements set the right way.
+
+## Configure packages
+> note: install the above packages first!
+
+### Postgres
+You need to bootstrap the database with this snippet in PSQL:
+
+```bash
+CREATE DATABASE molgenis;
+CREATE USER molgenis WITH PASSWORD 'molgenis';
+GRANT ALL PRIVILEGES ON SCHEMA molgenis TO molgenis;
+```
+
+In the ```pg_hba.conf``` You need to update to these lines:
+
+```bash
+# TYPE  DATABASE        USER            ADDRESS                 METHOD
+
+# "local" is for Unix domain socket connections only
+local   all             all                                     peer
+# IPv4 local connections:
+host    all             all             127.0.0.1/32            md5
+# IPv6 local connections:
+host    all             all             ::1/128                 md5
+# Allow replication connections from localhost, by a user with the
+# replication privilege.
+#local   replication     postgres                                peer
+#host    replication     postgres        127.0.0.1/32            ident
+#host    replication     postgres        ::1/128                 ident
+```
+
+In the `postgres.conf` you need to update these lines:
+
+```bash
+max_locks_per_transaction = 1024
+```
+
+Restart the service.
+
+### Elasticsearch
+Open ```elasticsearch.yml``` in the Elasticsearch config directory and set the following properties:
+
+```bash
+cluster.name: molgenis
+node.name: node-1
+indices.query.bool.max_clause_count: 131072
+```
+
+In the ``` /etc/security/limits.conf``` update this line:
+
+```bash
+elasticsearch  -  nofile  65536
+```
+
+Add this this file in ```/etc/sysctl.d/99-elasticsearch.conf``` with the following content.
+
+```bash
+vm.swappiness=1
+```
+Restart the service.
+
+### Configure Tomcat
+After you install Apache Tomcat you first need to install MOLGENIS before you start the service.
+
+### Configure HTTPD
+We use proxy passes to expose Tomcat through Apache. You can use this file as template to expose your MOLGENIS instance.
+Create the file in ```/etc/httpd/conf.d/```
+
+**molgenis.conf**
+
+```
+<VirtualHost molgenis.your-domain.ext:80>
+
+    ServerAdmin admin@molgenis.your-domain.ext
+
+    ServerName molgenis.your-domain.ext
+    ServerAlias molgenis.your-domain.ext
+
+    RewriteEngine on
+    RewriteCond %{SERVER_NAME} =molgenis.your-domain.ext
+    RewriteRule ^ https://%{SERVER_NAME}%{REQUEST_URI} [NE,R=permanent]
+
+</VirtualHost>
+
+<VirtualHost molgenis.your-domain.ext:443>
+
+    ServerAdmin admin@your-domain.ext
+
+    ServerName molgenis.your-domain.ext
+    ServerAlias molgenis.your-domain.ext
+
+    SSLProxyEngine on
+    RewriteEngine on
+
+    RewriteRule ^/@molgenis-ui/app-manager/(.+)         /@molgenis-ui/app-manager@0.1.4/$1 [P]
```

Comment:
Does the 'MOLGENIS release' standard operating procedure include steps to update this list?

Reply:
I checked, and yes it does